study1/week1/Third Maximum Number.java

```diff
@@ -0,0 +1,30 @@
+class Third Maximum Number {
```

Comment:
It's okay to keep the original class name from leetcode.
This will not be valid Java code. (To have it in the file name is enough)